content/posts/mvis/index.md

```diff
@@ -46,8 +46,16 @@ No control required
   - Click the icon again for `Nightcore Mod` : Uguuuuuuuu...
     - Increases speed by 1.50x and raises audio pitch.
 
+# music player
+
+You can open mvis music player without selecting a song in a song list and play just one song by going to mvis rulesets setting and click `Open Main Screen` button and mvis player will be appear. You can drag your mouse cursor to the right of a game screen to toggle rulesets setting and music player there.
+
+{{< youtube id="I0tl4Ira-v0" >}}
+
 # settings
 
+Open mvis music player by clicking `Open Main Screen` in mvis settings.
```

Comment:
`Open Main Screen` - Reveals the visualizer which can be used with the osu! track player.